Joe - Consolidate goals, transparent feedback (#3288)
PostHog/posthog.com: contents/handbook/company/team/joe-martin.md

## Current areas of focus
### Execution - Q1 2022
- [ ] **Community:** Organize the first PostHog community meet-up, with a good speaker. Retro the event and roll changes into a second event. Continue to attend and report back on other community events and meet-ups.
- [ ] **Case studies:** Secure an enterprise-level case study. Enhance the Customers section of the site and make plans to better leverage this content.
- [ ] **Content:** Support the Content team in creating a decent pipeline of good content that continues to deliver.
- [ ] **Partnerships:** Find further partnership opportunities to grow the community.
- [ ] **Internal:** Continue to advocate for less technical team members and users. Become more ambitious when it comes to shipping.

### Personal - Q1 2022
- [ ] Don't lose (another, again) mission in Pandemic Legacy: Season Zero.
- [ ] See all of my nieces at least once. And my nephew.
- [ ] Don't miss a Donut chat.

## Feedback
In our last [360 feedback session](/handbook/people/feedback#full-team-feedback-sessions-1), I received feedback on the following points. 

- I am seen as an effective and flexible executor, planner, writer and storyteller.

> - I am seen as empathetic and extremely supportive of colleagues, especially new colleagues or those with less experience than me. 

Author: ```suggestion
- I am seen as empathetic and extremely supportive of others, especially new colleagues or those with less experience than me. 
```

Reusing the same word that close together, Joe? Really? I THOUGHT BETTER OF YOU.

- I am seen as brave when giving feedback and stepping on toes. I default to transparency.
- Work I do, such as [case studies](/customers) is seen as valuable and showing a lot of insight.
- I am seen as over-cautious and too detail-orientated when planning new work. 

> - I am seen as over-communicative in social discussions and on the topic of diversity. 

Reviewer: All looks great to me, just this point - I had a different take on from the feedback session on diversity specifically. My point wasn't that you were over-communicative but more challenging you to turn that communication/discussion into further action. 

I think 'over-communicative' makes it sounds like you talk about diversity too much, which I certainly wouldn't want to discourage you from doing!

(If you are synthesizing other people's feedback here though and I've missed the point, please disregard - I'm biased towards my own feedback to you on this point.)

Author: Yeah, this is probably a fault of me trying to be too succint. Will update.

> - I am seen as over-communicative in social discussions and on the topic of diversity. 

Author: ```suggestion
- I am seen as over-communicative in social discussions. I talk a lot about diversity, but fail on follow-through.
```

> - I am seen as being biased to my past experiences. 

Author: ```suggestion
- I am seen as being biased to my past experiences. This limits my potential impact.
```

> - I am seen as being biased to my past experiences. This limits my potential impact.

Author: ```suggestion
- I am seen as biased to my past experiences. This limits my potential impact.
```

As a result, over the next six months I will:

- Push for bigger, braver projects. I feel continuing the '[Nail Plugins](https://github.com/PostHog/posthog.com/issues/3265)' project is a good starting point, as well as unblocking [competitor comparisons](https://github.com/PostHog/posthog.com/issues/2939).

> - Default to giving only high-level feedback on content, unless requested. 

Author: ```suggestion
- Default to giving only high-level feedback on content, unless requested to do more. 
```

> - Change my internal communication style to allow other voices more room, while also seeking opportunities to champion diversity at PostHog through action. 

Author: ```suggestion
- Change my internal communication style to allow other voices more room. Actively seek opportunities to champion diversity at PostHog through action. 
```
